include/sys/stat.h

```diff
@@ -0,0 +1,39 @@
+#ifndef _STAT_H
+#define _STAT_H
+
+#include <sys/types.h>
+
+struct stat {
```

Comment:
The right way to do this is to have a include/bits/stat.h that defines the stat struct, so the rest of this file can be automatically generated